README.adoc

```diff
@@ -64,9 +64,10 @@ kubectl create -f samples/echo-function.yaml
 
 == Watch Echo Function Sidecar Log
 
+In another terminal
 ```
-# In another terminal
-kubectl logs -f [echo-pod-name] -c sidecar
+#kubectl logs -f -c sidecar [echo-pod-name]
```

Comment:
did you intentionally leave this commented out line here?

Reply:
yes, but can remove